sdks/wasm/Mono.WebAssembly.DebuggerProxy/MonoProxy.cs

```diff
@@ -216,38 +216,34 @@ protected override async Task<bool> AcceptCommand (MessageId id, string method,
 
 			case "Runtime.getProperties": {
 					var objId = args? ["objectId"]?.Value<string> ();
-					if (objId.StartsWith ("dotnet:", StringComparison.Ordinal)) {
-						var parts = objId.Split (new char [] { ':' });
-						if (parts.Length < 3)
-							return true;
-						switch (parts[1]) {
-						case "scope": {
-							await GetScopeProperties (id, int.Parse (parts[2]), token);
-							break;
-							}
-						case "object": {
-							await GetDetails (id, MonoCommands.GetObjectProperties (int.Parse (parts[2]), expandValueTypes: false), token);
-							break;
-							}
-						case "array": {
-							await GetArrayDetails (id, objId, parts, token);
-							break;
-							}
-						case "valuetype": {
-							await GetDetailsForValueType (id, objId,
-									get_props_cmd_fn: () => {
-										if (parts.Length < 4)
-											return null;
-
-										var containerObjId = int.Parse (parts[2]);
-										return MonoCommands.GetObjectProperties (containerObjId, expandValueTypes: true);
-									}, token);
-							break;
-							}
-						}
+					if (!objId.StartsWith ("dotnet:", StringComparison.Ordinal))
+						// not for us
+						break;
 
+					var parts = objId.Split (new char [] { ':' });
+					if (parts.Length < 3) {
+						// FIXME: um return an error?
 						return true;
```

Comment:
```suggestion
						return false;
```
Can't return true without sending a response